### PR TITLE
Example Battle Issue #88

### LINKED ISSUE #88: Example code for starting a battle
We need to provide code examples for starting a new battle in the documentation. Preferably, we should have a minimal example in the readme, as well as an example or 2 in `examples_test.go`.

### PR DESCRIPTION
**Pull request by rdhadda123**
- Draft
- Test
- Fixed tests that checked meta last move
- Code for example battle

closes #88

### CODE REVIEW

**Comment by adapap** [comment]
Just a tip: in your issue text, you can write keywords such as closes or fixes #88, and it will automatically link that issue to this pull request (so that it closes when this is merged).

File: README.md [hunk 1]
```diff
@@ -9,6 +9,37 @@ A portable library for accurately simulating pokemon battles.
 
 [Documentation](https://pkg.go.dev/github.com/StevensSEC/pokemonbattlelib)
 
+func ExampleBattle() {
```
Comment (dyc3) [inline README.md:12]: needs to be inside a code block, with the language `go`

See: https://www.markdownguide.org/extended-syntax/#syntax-highlighting

**Review by dyc3 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ A portable library for accurately simulating pokemon battles.
 
 [Documentation](https://pkg.go.dev/github.com/StevensSEC/pokemonbattlelib)
 
+~~~go
```
Comment (dyc3) [inline README.md:12]: These are tildas, not backticks
```suggestion
```go
```

File: README.md [hunk 3]
```diff
@@ -40,6 +41,7 @@ func ExampleBattle() {
 	//Bulbasaur fainted
 }
 //Displaying how a battle works. Two Pokemon are created and keep battling and their stats change such as health, number of moves, etc.
+~~~
```
Comment (dyc3) [inline README.md:44]:
```suggestion
```
```

**Review by dyc3 — CHANGES_REQUESTED** [review]

**Comment by dyc3** [comment]
Also the tests are failing for this PR @rdhadda123

**Review by dyc3 — APPROVED** [review]
@rdhadda123 I went ahead and fixed up this PR for you. There was a few small issues, but they're fixed now.

**Pull request merged**